.vscode/settings.json

```diff
@@ -5,8 +5,20 @@
   "html.format.enable": false,
   // Disable the default javascript formatter
   "javascript.format.enable": false,
-  // Use 'prettier-eslint' instead of 'prettier'. Other settings will only be fallbacks in case they could not be inferred from eslint rules.
-  "prettier.eslintIntegration": true,
+  // Using the prettier-vscode plugin as the old eslint integration has been deprecated. More here: https://github.com/prettier/prettier-vscode/issues/870
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
```

Comment:
Is this configuration specifically for vs code users?

Reply:
I believe everything under the `.vscode/` folder is only applied by VSCode (maybe also some compatible forks?) But it should not affect other editors unless they purposefully read that file.